pkg/util.go

```diff
@@ -111,6 +113,23 @@ func MapsToJwkSet(maps []map[string]interface{}) (*jwk.Set, error) {
 	return set, nil
 }
 
+// IsJWK tests whether the given map (all) can is a parsable representation of a JWK. If not, an error is returned.
+// If nil is returned, all supplied maps are parsable as JWK.
+func IsJWK(maps ...interface{}) error {
```

Comment:
Is this the validateJWK func that has been moved. An 'IS' func would suggest a boolean return?

Reply:
It is. I'll rename it to ValidateJWK